### ISSUE TITLE
No 'version' key in the manifest file for custom integration 'dualmode_generic'. This will not be allowed in a future version of Home Assistant. Please report this to the maintainer of 'dualmode_generic'

### ISSUE THREAD

**Issue by towerclimber87** [zacs/ha-dualmodegeneric#19]
I am gettting this error. the device works fine but i am afraid that it will stop working.

**Comment by GsynthChris** [comment]
I just updated to the most recent HA and not the error specifies a date. Looks like it will be blocked in the June release

No 'version' key in the manifest file for custom integration 'dualmode_generic'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'dualmode_generic'

**Comment by GsynthChris** [comment]
Work around was pretty simple. I just added a line to the manifest.json with `"version":"1.0"` and after a restart the error isn't in my logs anymore. 

```
{
    "domain": "dualmode_generic",
    "name": "Dual-Mode Generic Thermostat",
    "documentation": "https://github.com/zacs/ha-dualmodegeneric",
    "version": "1.0",
    "dependencies": [],
    "codeowners": ["@zacs"],
    "requirements": []
}
```

**Comment by towerclimber87** [comment]
Awesome, thanks!

**Comment by franfos** [comment]
Hi @zacs ,
are you still on charge of this repo? Maybe, if you are not able to attend it, you can add a collaborator who can keep it alive.

Best regards.

**Comment by zacs** [comment]
I would be happy to add a collaborator, if someone is willing. Obviously the version addition is simple -- fixed in 8702384e528fb0d9f2d3ef21539a003b5574e31f.
